swagger-initializer.js

```diff
@@ -1,14 +1,15 @@
 window.onload = function() {
```

Comment:
docs will basically say this file is from our site, and the rest of things that aren't IDL are from swagger. capiche? ;)

basically it should be obvious how extremely simple this gh-pages branch is, and that it is entirely intentionally dumb. I'll take on this documentation